packages/cli/src/lib/epub/index.js

```diff
@@ -9,26 +9,28 @@ const __dirname = path.dirname(__filename)
  * A façade delegation module for EPUB libraries
  */
 export default async (lib = 'epubjs', options = {}) => {
-  let libPath
+  let libName, libPath
 
   switch (lib.toLowerCase()) {
     case 'epubjs': {
+      libName = 'Epub.js'
       libPath = path.join(__dirname, 'epub.js')
       break
     }
     case 'pandoc': {
+      libName = 'Pandoc'
       libPath = path.join(__dirname, 'pandoc.js')
       break
     }
     default:
-      console.error('[CLI:lib/epub] Unrecognized EPUB library ${lib}')
+      console.error(`[CLI:lib/pdf] Unrecognized EPUB library '${lib}'`)
```

Comment:
```suggestion
      console.error(`[CLI:lib/epub] Unrecognized EPUB library '${lib}'`)
```

Reply:
Ah, that is fixed on the `feature/quire-cli-epub-command` branch